collectors/deployments_collector.go

```diff
@@ -11,6 +11,7 @@ import (
 type DeploymentsCollector struct {
 	deploymentReleaseInfoMetric                *prometheus.GaugeVec
 	deploymentStemcellInfoMetric               *prometheus.GaugeVec
+	deploymentVMTypeCountMetric                *prometheus.GaugeVec
```

Comment:
Can you please remove `Count` from this variable name, it's implicit by the type.